src/modules/QtQml/Connections.js

```diff
@@ -0,0 +1,41 @@
+/* Connections */
+registerQmlType({
+  module: 'QtQml',
+  name: 'Connections',
+  versions: /.*/,
+  baseClass: 'QtObject',
+  constructor: function QMLConnections(meta) {
+    callSuper(this, meta);
+
+    createProperty("QtObject", this, "target");
+    createProperty("bool", this, "ignoreUnknownSignals");
+
+    this.target = this.$parent
+
+    var old_target = this.target;
```

Comment:
perhaps it should be moved closer to `reconnectTarget`? Also, it should be `let`.